gnoi_client/gnoi_client.go

```diff
@@ -62,6 +63,14 @@ func main() {
 		default:
 			panic("Invalid RPC Name")
 		}
+	case "File":
+		fc := gnoi_file_pb.NewFileClient(conn)
+		switch *rpc {
+		case "Stat":
+			fileStat(fc, ctx)
+		default:
+			panic("Invalid RPC Name")
```

Comment:
I don't think we should panic here.

Reply:
Thanks for the suggestion, I opened an ADO 29031150 to refactor gNOI client code as suggested. I will make these changes in a future PR to compartmentalize the change